inspiration-board/.gitignore

```diff
@@ -0,0 +1,23 @@
+# See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
```

Comment:
Note the paths of these files, you put an inspiration board project **inside** the inspiration board project.